### PR TITLE
Replace chef with ansible

### PR DESCRIPTION
**Pull request by jack-trikeapps**
The chef-solo provisioner has proven to be really brittle and hard to understand. Replace it with an ansible-powered one.

Porting this to eaforum is left as an exercise to the reader.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,75 +4,9 @@
...
-	url = https://github.com/opscode-cookbooks/yum-epel.git
-[submodule "provisioning/cookbooks/dmg"]
-	path = provisioning/cookbooks/dmg
-	url = https://github.com/opscode-cookbooks/dmg.git
-[submodule "provisioning/cookbooks/windows"]
-	path = provisioning/cookbooks/windows
-	url = https://github.com/opscode-cookbooks/windows.git
-[submodule "provisioning/cookbooks/chef_handler"]
-	path = provisioning/cookbooks/chef_handler
-	url = https://github.com/opscode-cookbooks/chef_handler.git
```
Comment (moggyboy) [inline .gitmodules]: Beautiful

File: Vagrantfile [hunk 2]
```diff
@@ -1,122 +1,13 @@
...
-  #   chef.validation_key_path = "ORGNAME-validator.pem"
-  # end
-  #
-  # If you're using the Opscode platform, your validator client is
-  # ORGNAME-validator, replacing ORGNAME with your organization name.
-  #
-  # If you have your own Chef Server, the default validation client name is
-  # chef-validator, unless you changed the configuration.
-  #
-  #   chef.validation_client_name = "ORGNAME-validator"
```
Comment (moggyboy) [inline Vagrantfile]: Is there anything more pleasing than gigantic swathes of red?

**Comment by moggyboy** [comment]
Looks glorious 👍

**Comment by jack-trikeapps** [comment]
Can you test it and merge if it works locally for you? Also the wiki will need updating.

**Pull request merged**